test/jdk/java/lang/management/ThreadMXBean/VirtualThreadDeadlocks.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Same as some other files with copyright year updates, should have been `2022, 2023,`